### ISSUE TITLE
Default value in docs for TerminalInteractiveShell.simple_prompt is incorrect

### ISSUE THREAD

**Issue by doolio** [ipython/ipython#14402]
The default value for `TerminalInteractiveShell.simple_prompt` per the [docs](https://ipython.readthedocs.io/en/stable/config/options/terminal.html#configtrait-TerminalInteractiveShell.simple_prompt) is `True` yet when one produces the default profile `ipython_config.py` the comments imply the default value is `False`.

In addition, the statement "and emacs inferior-shell integration through elpy." should be re-worded to "and emacs inferior-python integration." as "inferior" here in Emacs parlance just means the process is a subprocess but the process here is the ipython/python interpreter and it is also *not just* applicable through Elpy as it can be used ~~within~~ without Elpy. The 'inferior-shell' refers to the subprocess when using the Emacs built-in shell (M-x shell).

I would submit a PR but it is unclear to me if I would have to make changes in [all these places](https://github.com/search?q=repo%3Aipython%2Fipython%20TerminalInteractiveShell.simple_prompt&type=code).

Edit: changed within for without

**Comment by Carreau** [comment]
Ah interesting, 

this is because the default value with depends as to wether we are in a tty.

```
This mode default to `True` if the `IPY_TEST_SIMPLE_PROMPT`
environment variable is set, or the current terminal is not a tty
```

You can see that by using `ipython --help-all`, and `ipython --help-all| less`, both will report different default values. 

See #14404 to extend phrasing.

**Comment by doolio** [comment]
> See https://github.com/ipython/ipython/pull/14404 to extend phrasing.

Would you like me to extend the phrasing further?

**Comment by Carreau** [comment]
> Would you like me to extend the phrasing further?

Sure, especially wrt emacs, I do not use it so will trust emacs users for clarifications. 

I can also merge #14404 and you can send a subsequent PR as well, if that works for you.

**Comment by doolio** [comment]
> I can also merge https://github.com/ipython/ipython/pull/14404 and you can send a subsequent PR as well, if that works for you.

I think you may have to as I was not allowed to push commits to your PR (permission denied).

**Comment by Carreau** [comment]
> I think you may have to as I was not allowed to push commits to your PR (permission denied).

Yeah, the alternative is to make a PR against my branch, or just comment and I include the changes. But I just merge and will wait for your PR.

**Comment by doolio** [comment]
> and will wait for your PR.

Available as PR #14407.
